my-app/src/modules/Cart/Checkout/CheckoutSuccess.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link, useNavigate } from "react-router-dom";
+import { styled } from '@mui/system';
+import { Typography, Button } from '@mui/material';
+import handleLogout from "../Services/LogoutApi";
+import Header from "../../../Shared/Header/Header";
+import Footer from "../../../Shared/Footer/Footer";
+
+const Container = styled('div')({
```

Comment:
create a class in .css file move this style inside the class and use that class here.